mediaproviders/src/main/java/com/kaltura/playkit/providers/api/ovp/model/KalturaCaptionType.java

```diff
@@ -0,0 +1,19 @@
+package com.kaltura.playkit.providers.api.ovp.model;
+
+public enum KalturaCaptionType {
+    srt("1"),
+    dfxp("2"),
+    webvtt("3"),
+    cap("4");
+
+
+    private final String captionType;
+
+    private KalturaCaptionType(String captionType) {
+        this.captionType = captionType;
+    }
+
+    public String getCaptionType() {
```

Comment:
It's better to avoid storing this string/number as a field. Then getCaptionType() is just a `switch(this)`.